website/content/en/docs/Installation/_index.md

```diff
@@ -898,4 +898,15 @@ below is the full list of those volumeMounts:
 * jenkins-home
 * scripts
 * init-configuration
-* operator-credentials
\ No newline at end of file
+* operator-credentials
+
+## Validating Webhook 
+You can also have a validating webhook which can be used to validate security warnings in plugins. It can be easily installed via helm charts by setting webhook.enabled in values.yaml or in the command line flag.
```

Comment:
```suggestion
A validating webhook can be used in order to increase monitoring capabilities of the security issues. It will look for security vulnerabilities in the base and requested plugins. It can be easily installed via helm charts by setting webhook.enabled in values.yaml. If the installation happens with yamls, please provide additional fields:
```yaml
HERE OPERATOR YAML
```
```